clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/DataRecoveryClient.java

```diff
@@ -45,8 +52,35 @@ public DataRecoveryMonitor getMonitor() {
     return monitor;
   }
 
+  public void filterStoresWithOngoingRepush(Set<String> storesList, StoreRepushCommand.Params params) {
+    String url = params.getUrl();
+    ControllerClient cli = params.getPCtrlCliWithoutCluster();
+    LocalDateTime timestamp = params.getTimestamp();
+    String sourceFabric = params.getSourceFabric();
+    for (String s: storesList) {
+      String clusterName = cli.discoverCluster(s).getCluster();
+      try (ControllerClient parentCtrlCli = new ControllerClient(clusterName, url, Optional.empty())) {
+        StoreHealthAuditResponse storeHealthInfo = parentCtrlCli.listStorePushInfo(s, false);
+        RegionPushDetails regionPushDetails = storeHealthInfo.getRegionPushDetails().get(sourceFabric);
+        String latestTimestamp = regionPushDetails.getPushStartTimestamp();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSS");
```

Comment:
Can we use the predefined constant [DateTimeFormatter.ISO_LOCAL_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) in this case?